test/test_functional.py

```diff
@@ -191,6 +191,76 @@ def test_istft_of_sine(self):
         self._test_istft_of_sine(amplitude=80, L=9, n=6)
         self._test_istft_of_sine(amplitude=99, L=10, n=7)
 
+    def _test_linearity_of_istft(self, kwargs):
+        for data_size in self.data_sizes:
+            for i in range(self.number_of_trials):
+                sound1 = common_utils.random_float_tensor(i, data_size)
+                sound2 = common_utils.random_float_tensor(i * 2, data_size)
+                a, b = torch.rand(2)
+                stft1 = torch.stft(sound1, **kwargs)
+                stft2 = torch.stft(sound2, **kwargs)
+                istft1 = torchaudio.functional.istft(stft1, **kwargs)
+                istft2 = torchaudio.functional.istft(stft2, **kwargs)
+                sound = a * istft1 + b * istft2
+                estimate = torchaudio.functional.istft(a * stft1 + b * stft2,  **kwargs)
```

Comment:
Thanks for looking into this! Let's avoid using STFT in this test, and generate a random `specgram` tensor by generating directly a tensor of the form of the output of STFT.